### DISCUSSION TITLE
Outstanding pull requests

### DISCUSSION THREAD

**Discussion by chumanfu** [netsuitephp/netsuite-php#145] [General]
Hello,

There are 2 outstanding pull requests which would be great to get into this project.
Does anyone know when these will be reviewed and merged? Both add great value to the library.

**Comment by neclimdul** [comment]
Yeah, I need to finish up the logging changes.

I'm less sure about the retry support. The current design seems like it would just stampede an API that's not working. I think generally a better solution is to implement the actual logic in the system using the library. For-example using laravel jobs to queue a later request.

**Comment by chumanfu** [comment]
Thanks for your reply. That makes sense :-)

**Comment by jrebs** [comment]
My job has been quite busy lately so I have been putting off the two issues in question for a time when I have a clear chunk of time to dedicate to making decisions and testing stuff out (it's non-trivial to test new code when you don't have a personal netsuite account).

That being said, I was already leaning strongly to the same position as @neclimdul on the matter of the automatic retry functionality. My immediate thought was that it is outside of the realm of responsibility for this library.

**Reply by neclimdul** [reply]
maybe we just close that then :laughing:
